android/src/main/java/io/rownd/android/models/domain/AppConfig.kt

```diff
@@ -47,11 +47,30 @@ enum class AppSchemaEncryptionState {
 @Serializable
 data class AppConfigConfig constructor(
     val hub: HubConfig = HubConfig(),
+    val customizations: CustomizationsConfig = CustomizationsConfig()
+)
+
+@Serializable
+data class CustomizationsConfig @OptIn(ExperimentalSerializationApi::class) constructor(
+    @SerialName("primary_color")
+    @JsonNames("primaryColor")
```

Comment:
Do we need the alternate `JsonNames` for this?

Reply:
JsonNames have been removed